.github/workflows/ci.yml

```diff
@@ -154,7 +154,7 @@ jobs:
       uses: actions/checkout@v4
       with:
         repository: data-apis/array-api-tests
-        ref: '33f2d2ea2f3dd2b3ceeeb4519d55e08096184149'  # Latest commit as of 2024-05-29
+        ref: 'd295a0a66cd82a43e84c1b8d73ca198cc45e9d23'  # Latest commit as of 2024-05-29
```

Comment:
Please update comment to: `Latest commit as of 2024-06-10`

Reply:
Done in https://github.com/pydata/sparse/pull/696/commits/42c9bfd1e7bfbed42550ceec3d803d6a53b3497b.